client/extensions/woocommerce/woocommerce-services/views/carrier-accounts/index.js

```diff
@@ -0,0 +1,107 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import { some } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import ExtendedHeader from 'woocommerce/components/extended-header';
+import CarrierAccountListItem from './list-item';
+import * as CarrierAccountsActions from '../../state/carrier-accounts/actions';
+import { getSelectedSiteId } from 'state/ui/selectors';
+
+class CarrierAccounts extends Component {
+	renderListHeader = carriers => {
+		const { translate } = this.props;
+
+		return (
+			<div className="carrier-accounts__header">
+				<div className="carrier-accounts__header-icon" />
+				<div className="carrier-accounts__header-name">{ translate( 'Name' ) }</div>
+				{ some(carriers, 'credentials' ) && <div className="carrier-accounts__header-credentials">{ translate( 'Credentials' ) }</div>}
+
+				<div className="carrier-accounts__header-actions" />
+			</div>
+		);
+	};
+
+	renderListItem = ( carrier, index ) => {
+		const { siteId, isFetching } = this.props;
+
+		return (
+			<CarrierAccountListItem
+				key={ index }
+				siteId={ siteId }
+				isPlaceholder={ isFetching }
+				data={ carrier }
+			/>
+		);
+	};
+
+	render() {
+		const { translate } = this.props;
+
+		const onConnect = ( carrier ) => {
+			const url = new URL( window.location.href );
+			url.searchParams.set( 'carrier', carrier.carrierId );
+			window.location.href = url.href;
+		}
+
+		const onDisconnect = () => {
+		}
+
+		const carriers = [
+			{ carrierId: 'UPS', name: 'UPS', onConnect },
+			{ carrierId: 'UPS', name: 'UPS', credentials: '989999847463', onDisconnect },
+		]
```

Comment:
Are we leaving the actual registration API calls out from this PR?

Reply:
Yeah, sorry I should've mentioned it in the PR's description.

Reply:
Preferably to leave the `credentials` blank than to have a value.